cli/cli/cli.go

```diff
@@ -41,7 +42,7 @@ import (
 	tui "github.com/opencurve/curveadm/internal/tui/common"
 	"github.com/opencurve/curveadm/internal/utils"
 	cliutil "github.com/opencurve/curveadm/internal/utils"
-	log "github.com/opencurve/curveadm/pkg/log/glg"
+	"github.com/opencurve/curveadm/pkg/log/zaplog"
```

Comment:
```suggestion
	log "github.com/opencurve/curveadm/pkg/log/zaplog"
```
This keeps your changes small.

Reply:
done